hangman.py

```diff
@@ -0,0 +1,356 @@
+    print("You have", warnings_point,"warnings left")
+    while guesses_point>0:
+        print("-------")
+        print("You have", guesses_point, "guesses left")
+        print("Available letters:", end=" ")
+        get_available_letters(letters_guessed)
+        letter=input("Please guess a letter:")
+        if str.isalpha(letter.lower())==False or len(letter)!=1:
+            warnings_point-=1
+            print("Oops! That is not a valid letter or you print more than one letter.",end=" ")
+            print("You have", warnings_point, "warnings left:",end=" ")
+            print(get_guessed_word(secret_word, letters_guessed))
+        elif letter in letters_guessed:
+            warnings_point-=1
+            print("You have already guessed that letter:",end=" " )
+            print("You have", warnings_point,"warnings left:")
+            print(get_guessed_word(secret_word, letters_guessed))
+            if warnings_point<0:
+                guesses_point-=1
+                print("Oops! You have no warnings left, so you lose ine guess:",end=' ')
+                get_guessed_word(secret_word,letters_guessed)
+        else:
+            if letter in secret_word:
+                letters_guessed.append(letter)
+                print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+            else:
+                letters_guessed.append(letter)
+                if letter in "aeoiu":
+                    guesses_point-=2
+                else:
+                    guesses_point-=1
+                print("Oops! That letter is not in my word:", get_guessed_word(secret_word,letters_guessed))
+        if is_word_guessed(secret_word,letters_guessed)==0:
+            final_points=guesses_point*int(len(secret_word))
+            print("Congratulations, you won!")
+            print("Your total score for this game is:",final_points)
+            guesses_point=-1
+        if guesses_point==0:
+            print("-------")
+            print("Sorry, you ran out of guessed. The word was", secret_word)
+            break
+
+
+
+# When you've completed your hangman function, scroll down to the bottom
+# of the file and uncomment the first two lines to test
+#(hint: you might want to pick your own
+# secret_word while you're doing your own testing)
+
+
+# -----------------------------------
+
+
+
+def match_with_gaps(my_word, other_word):
+    '''
+    my_word: string with _ characters, current guess of secret word
+    other_word: string, regular English word
+    returns: boolean, True if all the actual letters of my_word match the 
+        corresponding letters of other_word, or the letter is the special symbol
+        _ , and my_word and other_word are of the same length;
+        False otherwise: 
+    '''
+    my_word1 = "".join(my_word.split(" "))
+    if len(my_word1) == len(other_word):
+        for i in range(len(my_word1)):
+            if my_word1[i] == '_':
+                continue
```

Comment:
If you do nothing here - then just remove that code. You need to have here only that `if` statement where you do a `return`